apps/nextjs-website/src/app/theme.ts

```diff
@@ -1,6 +1,6 @@
 'use client';
 import { createTheme } from '@mui/material/styles';
-import { theme as MUIItaliaTheme } from '@pagopa/mui-italia';
+import { theme as MUIItaliaTheme } from '../../../../node_modules/@pagopa/mui-italia/dist/theme';
```

Comment:
```suggestion
import { theme as MUIItaliaTheme } from '@pagopa/mui-italia';
```
I'm assuming this was an old change?